### DISCUSSION TITLE
How to implement the function of the "unevalutedProperties" annotation?

### DISCUSSION THREAD

**Discussion by manchuanxingmengyaxinhe** [victools/jsonschema-generator#268] [Q&A] [answered]
i want to implment "unevalutedProperties" through  @Schema, but i don't find this in you vitools doc . What should I do

**Comment by CarstenWickner** [comment, marked as answer]
Hi @manchuanxingmengyaxinhe,

As per your other question, you're already making use of the "instance attribute override" feature, which is likely a good approach here as well, short of a "custom definition". But that might give you trouble in the case of recursive structures, i.e., the attribute overrides might be the "safer" option.
